bundles/org.openhab.binding.tr064/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,131 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="tr064"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<bridge-type id="generic">
+		<label>Generic CPE</label>
+
+		<config-description>
+			<parameter name="host" type="text" required="true">
+				<label>Host</label>
+				<description>Host name or IP address.</description>
+				<context>network-address</context>
+			</parameter>
+			<parameter name="user" type="text">
+				<label>Username</label>
+				<default>dslf-config</default>
+			</parameter>
+			<parameter name="password" type="text" required="true">
+				<label>Password</label>
+				<context>password</context>
+			</parameter>
+			<parameter name="refresh" type="integer">
+				<label>Refresh Interval</label>
+				<default>60</default>
+				<unitLabel>s</unitLabel>
+			</parameter>
+		</config-description>
+	</bridge-type>
+
+	<bridge-type id="fritzbox">
+		<label>Fritzbox</label>
+		<description>A physical Fritzbox Device.</description>
+
+		<config-description>
+			<parameter name="host" type="text" required="true">
+				<label>Host</label>
+				<description>Host name or IP address.</description>
+				<context>network-address</context>
+			</parameter>
+			<parameter name="user" type="text">
+				<label>Username</label>
+				<default>dslf-config</default>
+			</parameter>
+			<parameter name="password" type="text" required="true">
+				<label>Password</label>
+				<context>password</context>
+			</parameter>
+			<parameter name="refresh" type="integer">
+				<label>Refresh Interval</label>
+				<default>60</default>
+				<unitLabel>s</unitLabel>
+			</parameter>
+			<parameter name="tamIndices" type="text" multiple="true">
```

Comment:
I'd suggest to mark all parameters from here to the end as "advanced" to keep the user focused on the really relevant ones for the start.